config/locales/en/apply/student.yml

```diff
@@ -11,61 +11,54 @@ en:
         As a member, you will join a family of fellow students working to use our education
         and skills to make the world a better place.
       applications_closed: >
-        Our application is now closed for the Fall 2021 semester. We will be recruiting again next semester! Sign up for our mailing list to stay up to date.
+        Our application will open for the Spring 2022 semester on Wednesday, January 19th and close on Friday, January 28th at 11:59 PM PDT. Sign up for our mailing list or follow our facebook page to stay up to date!
```

Comment:
if it's possible, can you make 'facebook page' hyperlinked? https://www.facebook.com/CalBlueprint/